### PR TITLE
Name message fields

### LINKED ISSUE #1783: Improve error message
## Description

I think this error happens if something is provided as `undefined`. We should return a better error message in these cases, maybe need to check some values for undefined before calling the message interface
```
node examples/21-claim-outputs.js
Error:  {
  type: 'jsonError',
  error: 'missing field `payload` at line 1 column 31'
}
```
## Are you planning to do it yourself in a pull request?

Maybe.

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Name message fields to get better error messages when something is undefined

## Links to any relevant issues

Fixes #1783 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Running binding examples

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/lib/AccountManager.ts [hunk 1]
```diff
@@ -305,7 +305,7 @@ export class AccountManager {
     async setClientOptions(options: ClientOptions): Promise<void> {
         await this.messageHandler.sendMessage({
             cmd: 'setClientOptions',
-            payload: options,
+            payload: { clientOptions: options },
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/AccountManager.ts]: Stressing me out, can we just rename options :trollface: ?

File: bindings/python/iota_wallet/account.py [hunk 2]
```diff
@@ -10,7 +10,9 @@ def __init__(self, alias_index, handle):
     def __str__(self):
         message = {
             'cmd': 'getAccount',
-            'payload': self.alias_index,
+            'payload': {
+                'accountId': self.alias_index,
```
Comment (thibault-martinez) [inline bindings/python/iota_wallet/account.py]: Just curious, alias index? That's a tad confusing

Comment (Thoralf-M) [reply bindings/python/iota_wallet/account.py]: Bing did it like this, should I rename to accountId?

File: bindings/python/iota_wallet/wallet.py [hunk 3]
```diff
@@ -181,8 +194,10 @@ def set_stronghold_password(self, password):
...
         return self._send_cmd_routine(
-            'setStrongholdPassword',
-            password
+            'setStrongholdPassword', {
+                'password': password
+            }
+
         )
 
     def set_stronghold_password_clear_interval(self, interval_in_milliseconds):
```
Comment (thibault-martinez) [inline bindings/python/iota_wallet/wallet.py:203]: It already had a named field lol ?

File: bindings/python/iota_wallet/wallet.py [hunk 4]
```diff
@@ -198,8 +213,10 @@ def store_mnemonic(self, mnemonic):
         """Store mnemonic.
         """
         return self._send_cmd_routine(
-            'storeMnemonic',
-            mnemonic
+            'storeMnemonic', {
+                'mnemonic': mnemonic
+            }
+
         )
```
Comment (thibault-martinez) [inline bindings/python/iota_wallet/wallet.py:220]: Commenting at a random place: do we not have EmitTestEvent in python ?

File: bindings/python/iota_wallet/account.py (hunk 2 above)
Comment (thibault-martinez) [reply bindings/python/iota_wallet/account.py]: I would yeah, another PR if you prefer

File: bindings/nodejs/lib/AccountManager.ts (hunk 1 above)
Comment (samuel-rufi) [reply bindings/nodejs/lib/AccountManager.ts]: makes sense

File: bindings/python/iota_wallet/wallet.py (hunk 3 above)
Comment (Thoralf-M) [reply bindings/python/iota_wallet/wallet.py:203]: It was ahead of its time :trollface:

File: bindings/python/iota_wallet/wallet.py (hunk 4 above)
Comment (Thoralf-M) [reply bindings/python/iota_wallet/wallet.py:220]: No

**Review by thibault-martinez — COMMENTED** [review]
For the bindings, it's all internal, right ? Nothing breaking that needs a changelog ?

**Comment by Thoralf-M** [comment]
> Nothing breaking that needs a changelog ?

For Java and NodeJS yes, for Python not since also something new was added, but we had no release yet

**Review by thibault-martinez — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**